src/game.js

```diff
@@ -0,0 +1,117 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
+    const tilesWin = [];
+    this.tilesWin = tilesWin;
+    this.length = size * size;
+   
+    this.bindMethods();
+  
+  }
+  getWinnerTiles() {
+    for (let i = 0; i < this.length - 1; i++) {
+      this.tilesWin[i] = i + 1;
+    }
+    this.tilesWin[this.length - 1] = '';
+
+  }
+
+
+  bindMethods() {
+   
+    this.container = document.createElement('DIV');
+    this.getWinnerTiles = this.getWinnerTiles.bind(this);
+    this.renderTiles = this.renderTiles.bind(this);
+    this.beginGame = this.beginGame.bind(this);
+    this.shuffle = this.shuffle.bind(this);
+    this.moveToEmpty = this.moveToEmpty.bind(this);
+    this.checkTheEndOfGame = this.checkTheEndOfGame.bind(this);
+  }
+
+  init() {
+    const gameContainer = document.querySelector('.game-begin');
+    const startGame = document.createElement('BUTTON');
+    startGame.innerHTML = 'Start new Game';
+    startGame.classList.add('start-game');
+
+    this.container.classList.add('container');
```

Comment:
what is difference between `gameContainer` and `this.container`?

Reply:
gameContainer is consist of button "Start new Game" and this.container (here is we are rendering the tiles).

Reply:
Again the naming is confusing, that's why I ask.  
`this.container` is not clear name if you mean the game board